hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieInputFormatUtils.java

```diff
@@ -87,9 +87,17 @@ public static FileInputFormat getInputFormat(HoodieFileFormat baseFileFormat, bo
     }
   }
 
-  public static String getInputFormatClassName(HoodieFileFormat baseFileFormat, boolean realtime, Configuration conf) {
-    FileInputFormat inputFormat = getInputFormat(baseFileFormat, realtime, conf);
-    return inputFormat.getClass().getName();
+  public static String getInputFormatClassName(HoodieFileFormat baseFileFormat, boolean realtime) {
+    switch (baseFileFormat) {
+      case PARQUET:
```

Comment:
this is going to fail now, probably given we have a HFile base now. @garyli1019 can you please rebase and fix.

Reply:
thanks for letting me know